azure-pipelines.vrt-baseline.yml

```diff
@@ -1,88 +0,0 @@
-# Starter pipeline
```

Comment:
This file shouldn't be deleted

Reply:
should append the code below to this file instead:
https://github.com/microsoft/fluentui/blob/3287714774098eb5f902d9110a214033807d5e34/azure-pipelines.vrt-baseline-web-components.yml#L18-L42